adaptlearning/adapt_authoring
# orphaned element

fixes #1986 

## to test:
Import the demo course and change an elements _parentId or _id.
For instance if you change an articles _id attribute, it gets imported but all it's blocks and components are not imported. The id's of this elements should be listed in the frontends server log.
Inspect the imported course in the authoring tool. It should have created the page and the article without the blocks and components.
